gor-purdue: Success to move one probe to another probe

tony-sanguk opened the pull request:
I've just succeeded to move one unit to another place.

In detail, I made a probe to MOVE to another probe.
I could check the _ability id_ and _unit type tag_ from requesting data. Then I could find the MOVE ability has _ability_id: 16_ and the probe has the _unit_type_tag: 84_. In addition, I needed the one more thing, unit_tag which help us to distinguish each other units. I just used observation request, and I got the 12 probes unit_tag. I saved them, and used it to make a command MOVE. I checked the request is working well.

So, please check this code is running on your computer. But you might edit your map path.

tony-sanguk committed:
gor-purdue: Success to move one probe to another probe

gemblerz commented:
We don't want to make two versions of `core` and `sc2_comm`. Ideally, examples use the them.

```bash
import sys
sys.path.append('./core')
import core
```

tony-sanguk committed:
gor-purdue: Success to move one probe to another probe

- Delete examples/core.py
- Delete examples/sc2_comm.py
- Edit game.py that import core dir
